scripts/contentScript.js

```diff
@@ -5,6 +5,48 @@ const keyToLabel = Object.freeze({
 	'important': "Important"
 })
 
+async function cssForGithubTrackRepo(trackedRepos) {
+	const allOrgRepo = document.getElementById('org-repositories');
+	const ignoreUrl = [
+		'stargazers',
+		'forks',
+		'issues',
+		'pulls',
+		'issues?q=label%3A%22help+wanted%22+is%3Aissue+is%3Aopen',
+		'repositories?type=all',
+		'issues?q=label%3A%22Help+wanted%22+is%3Aissue+is%3Aopen'
+	];
+	const orgRepoUrl = Array.from(allOrgRepo.getElementsByTagName('a'));
+
+	orgRepoUrl.forEach((item) => {
+		const link = item.getAttribute('href').split('/');
+		const orgRepoName = link[link.length - 1];
+
+		if (ignoreUrl.includes(orgRepoName)) {
+		} else {
```

Comment:
```suggestion
		if (!ignoreUrl.includes(orgRepoName)) {
```

Why do you want to keep an empty block?